tests/softwareupdate/test_cloud_config.py

```diff
@@ -205,6 +197,7 @@ def test_cloud_config_buster_online(self, device_info_mock):
             None
         """
         self.check_if_githubapi_rate_limit_exceeded()
+        self.maxDiff = None
```

Comment:
what is that?

Reply:
this is needed for the testing to show the complete diff if it is to long